### DISCUSSION TITLE
下载和合并是在内存中完成的吗？

### DISCUSSION THREAD

**Discussion by cyfex** [xifangczy/cat-catch#178] [Q&A] [answered]
你好，我对js不是很熟悉，但我初步看来，ts的下载与合并成mp4都是在内存中完成的吧？因为我有一张固态硬盘，比较担心硬盘的写入量。如果是内存中完成的话，那实在是完美。

**Comment by xifangczy** [comment, marked as answer]
是的 内存中进行，最后完整视频才会推给浏览器下载。所以对内存大小有比较高的要求，否则无法处理下载大视频。
